Mod2/Preparation/Week4/IntroToORM.md

```diff
@@ -0,0 +1,17 @@
+## Pre-Work for [Intro To ORM](/Mod2/Lessons/Week4/IntroToORM.md)
+
+Tomorrow we are going to learn how to connect to a database from our console application. We're going to learn quite a few new vocab words. To prepare for tomorrow, review the database vocabulary we have learned so far. 
+
+Reach out to your group or an instructor if you're stuck coming up with one of these definitions!
+
+1. What is a database?
+2. What is SQL?
+3. What is PostgreSQL?
+4. What is a database table?
+5. What is a database row?
+6. What is a database column?
+7. What is a primary key?
+8. What is a foreign key?
+
+
+<!-- Any ideas for starting to think about ORM, EF, connecting to database type topics. All the videos and resources I'm finding are really in depth. -->
```

Comment:
Is there a snippet in any of the videos or resources that hits home? We can point to that specific section and say "read this only" or "view from 1:45 to 2:30", for example.

It might not be a bad thing that they're so in-depth, because it shows that these are concepts in which they will expand on, and we can point to different sections of these same resources for future lessons/labs/preps.

Reply:
I decided to point students to the official documentation intro page for EF Core, and mentioned that probably a lot of it doesn't make sense yet but to try and get a rough idea of the purpose of entity framework and how it fits in with what we've learned so far. https://learn.microsoft.com/en-in/ef/core/